<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIPortsPrefs</class>
 <widget class="PrefWidget" name="UIPortsPrefs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>663</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelDesc">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;This is a list of custom ports. Select &lt;span style=&quot; font-weight:700;&quot;&gt;RFC2217&lt;/span&gt; for remote serial port and &lt;span style=&quot; font-weight:700;&quot;&gt;SIGROK&lt;/span&gt; for using external sigrok-cli tool. Usage below:&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;RFC2217&lt;/span&gt; port: &amp;lt;hostname|ip&amp;gt;:&amp;lt;port&amp;gt; e.g. localhost:4000&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;SIGROK&lt;/span&gt; port: sigrok string, see &lt;a href=&quot;https://sigrok.org/wiki/Supported_hardware#Multimeters&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;Sigrok Multimeters&lt;/span&gt;&lt;/a&gt;e.g. &lt;span style=&quot; font-family:'monospace';&quot;&gt;uni-t-ut61e-ser:conn=/dev/ttyUSB0&lt;br /&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="2" column="0">
      <widget class="QComboBox" name="ui_customPortType0">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentText">
        <string notr="true">RFC2217</string>
       </property>
       <item>
        <property name="text">
         <string notr="true">RFC2217</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">SIGROK</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="ui_customPort0">
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QComboBox" name="ui_customPortType1">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentText">
        <string notr="true">RFC2217</string>
       </property>
       <item>
        <property name="text">
         <string notr="true">RFC2217</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">SIGROK</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="ui_customPort1">
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QComboBox" name="ui_customPortType2">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentText">
        <string notr="true">RFC2217</string>
       </property>
       <item>
        <property name="text">
         <string notr="true">RFC2217</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">SIGROK</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="ui_customPort2">
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QComboBox" name="ui_customPortType3">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentText">
        <string notr="true">RFC2217</string>
       </property>
       <item>
        <property name="text">
         <string notr="true">RFC2217</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">SIGROK</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="ui_customPort3">
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QComboBox" name="ui_customPortType4">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentText">
        <string notr="true">RFC2217</string>
       </property>
       <item>
        <property name="text">
         <string notr="true">RFC2217</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">SIGROK</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="ui_customPort4">
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QComboBox" name="ui_customPortType5">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentText">
        <string notr="true">RFC2217</string>
       </property>
       <item>
        <property name="text">
         <string notr="true">RFC2217</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">SIGROK</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="ui_customPort5">
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QComboBox" name="ui_customPortType6">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentText">
        <string notr="true">RFC2217</string>
       </property>
       <item>
        <property name="text">
         <string notr="true">RFC2217</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">SIGROK</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="ui_customPort6">
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QComboBox" name="ui_customPortType7">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentText">
        <string notr="true">RFC2217</string>
       </property>
       <item>
        <property name="text">
         <string notr="true">RFC2217</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">SIGROK</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="ui_customPort7">
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QComboBox" name="ui_customPortType8">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentText">
        <string notr="true">RFC2217</string>
       </property>
       <item>
        <property name="text">
         <string notr="true">RFC2217</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">SIGROK</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="ui_customPort8">
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QComboBox" name="ui_customPortType9">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentText">
        <string notr="true">RFC2217</string>
       </property>
       <item>
        <property name="text">
         <string notr="true">RFC2217</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">SIGROK</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="ui_customPort9">
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelType">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="labelPort">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelExe">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Sigrok executable</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ui_sigrokExe">
       <property name="text">
        <string notr="true">sigrok-cli</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="ui_sigrokExeButton">
       <property name="text">
        <string notr="true">...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="Spacer1_8">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PrefWidget</class>
   <extends>QWidget</extends>
   <header>prefwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="global">prefwidget.h</include>
 </includes>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>on_ui_sigrok_btn_clicked()</slot>
 </slots>
</ui>
